src/bundle/Resources/public/js/scripts/helpers/request.helper.js

```diff
@@ -4,7 +4,7 @@ const defaultGetErrorMessage = (error = {}) => error.errorMessage;
 
 const getErrorMessageObject = (response) => {
     const responseErrorMessage = response.json().then((jsonResponse) => {
-        return jsonResponse.ErrorMessage;
+        return jsonResponse.ErrorMessage ?? jsonResponse;
```

Comment:
With HTTP error code other than 2XX, `response.ok` below will be `false`.

I would treat `jsonResponse.ErrorMessage` not being set as an error in this function, and throw an `Error`. Otherwise you return an `object` instead of `string`, and there is a high likelihood that follow up code will fail on that instead.